configure.ac

```
@@ -202,6 +202,61 @@ AS_CASE([$with_gc],
     )
 AC_MSG_RESULT([$with_gc])
 
+dnl
+dnl User setting: native thread-local storage (off by default)
+dnl
+
+AC_ARG_WITH([native-tls],
+    [AS_HELP_STRING([--with-native-tls],
+        [use native thread-local storage implementation])],
+        [],
+        [with_native_tls=no])
+AC_MSG_CHECKING([whether to use native tls])
```

"whether to use native tls... pthread" is an odd message. Perhaps this:
```suggestion
AC_MSG_CHECKING([which TLS implementation to use])
```

Then down below, replace `AC_MSG_RESULT([yes])` by `AC_MSG_RESULT([native])`

---

Sounds good.